### DISCUSSION TITLE
Maybe a little of topic, but how to make a Gif into code?

### DISCUSSION THREAD

**Discussion by JesusGod-Pope666-Info** [Adrianotiger/desktopPet#135] [General]
@Adrianotiger You know how to make Gif into code.... You use it in your application correct?

I am know trying to get an app for FaviCon working on browsers with animation. Yet one of the scripts I have found needs the image in code....

You know how to make the image into the code? I just can't seem to find it on first sight.... Like.... Hack the Planet all the time, lets not just make it easy for users to do whatever they want on there website, easily and straightforward.

Thanks

Firefox works with Gif as Favicon, but not the branch of Google, like Opera. I found your 2 icons of the Sheep - the Gif and the ico file. And just want it to work the same on both. There was the cursor site I had, that did something of the same animation thing, but can't seem to find it at the moment.

-- Darkijah

**Comment by JesusGod-Pope666-Info** [comment]
![favicon](https://github.com/Adrianotiger/desktopPet/assets/90806094/f6b1265e-d265-40a8-ac50-36df4eb2efe7)

Apparently I can only upload one of them.... Wait a minute, Zip zip zip BRB

**Comment by JesusGod-Pope666-Info** [comment]
[FaviCon-Sheep.zip](https://github.com/user-attachments/files/16179921/FaviCon-Sheep.zip)

**Comment by JesusGod-Pope666-Info** [comment]
Might have another way. Sigh. All the ongoing nonsense of evil in this world.

**Comment by Adrianotiger** [comment]
> @Adrianotiger You know how to make Gif into code.... You use it in your application correct?
> 
> I am know trying to get an app for FaviCon working on browsers with animation. Yet one of the scripts I have found needs the image in code....
> 
> You know how to make the image into the code? I just can't seem to find it on first sight.... Like.... Hack the Planet all the time, lets not just make it easy for users to do whatever they want on there website, easily and straightforward.
> 
> Thanks
> 
> Firefox works with Gif as Favicon, but not the branch of Google, like Opera. I found your 2 icons of the Sheep - the Gif and the ico file. And just want it to work the same on both. There was the cursor site I had, that did something of the same animation thing, but can't seem to find it at the moment.
> 
> -- Darkijah

No, the application does not use any GIFs. It use a sprite sheet and cuts every image out from this sprite sheet.
You can use a gif as icon in the browser, but the application creates the animations from the sprite sheet and the xml file.

There is an editor to download for windows. If you open an animation.xml file, you should see the sprite sheet image.

**Reply by JesusGod-Pope666-Info** [reply]
Yea I am on Linux at the moment, and my keyboard for the windows 7 system I have died by water. Not sure I can repair it, although have surprisingly succeeded once. I think the keyboard might be 20 years or something..... I already got it after it was discarded from my earthly father long ago, at that time already looking very dirty and used, but hey, as long as it works and is/was functional then why throw it out. Usually keyboards don't last so long, as a liquid slip tends to happen here and there. Although in my younger years it also happened from losing in a game.
